src/ResourceManager/Sql/Commands.Sql/ServerKeyVaultKey/Services/AzureSqlServerKeyVaultKeyAdapter.cs

```diff
@@ -84,13 +84,10 @@ public IList<AzureSqlServerKeyVaultKeyModel> List(string resourceGroupName, stri
         /// <returns>The updated server key Vault Key model</returns>
         public AzureSqlServerKeyVaultKeyModel CreateOrUpdate(AzureSqlServerKeyVaultKeyModel model)
         {
-            var resp = Communicator.CreateOrUpdate(model.ResourceGroupName, model.ServerName, model.ServerKeyName, new ServerKeyCreateOrUpdateParameters()
+            var resp = Communicator.CreateOrUpdate(model.ResourceGroupName, model.ServerName, model.ServerKeyName, new ServerKey()
```

Comment:
This is a breaking change, so make sure to increment the version number appropriately